src/main/java/org/cbioportal/persistence/StudyViewRepository.java

```diff
@@ -1,9 +1,6 @@
 package org.cbioportal.persistence;
 
-import org.cbioportal.model.AlterationCountByGene;
-import org.cbioportal.model.ClinicalData;
-import org.cbioportal.model.ClinicalDataCount;
-import org.cbioportal.model.Sample;
+import org.cbioportal.model.*;
```

Comment:
@haynescd i think you said .* was against the rules right? IDE does this.

Reply:
Yea, I can show you the ide settings